src/mpi/request/waitall.c

```diff
@@ -153,11 +153,12 @@ int MPIR_Waitall_impl(int count, MPIR_Request * request_ptrs[], MPI_Status array
   Setting statuses, freeing requests objects, etc.
 */
 #undef FUNCNAME
-#define FUNCNAME MPIR_Waitall_post
+#define FUNCNAME do_waitall_post
```

Comment:
Maybe we should squash this into the waitall patch?

Reply:
done.